src/NodaTime.Test/LocalDateTest.cs

```diff
@@ -71,5 +71,22 @@ public void MinIsoValue()
             Assert.AreEqual(CalendarSystem.Iso, value.Calendar);
             Assert.Throws<OverflowException>(() => value.PlusDays(-1));
         }
+
+        [Test]
+        public void Deconstruction()
+        {
+            var value = new LocalDate(2017, 11, 7);
+            var expectedYear = 2017;
+            var expectedMonth = 11;
+            var expectedDay = 7;
+            value.Deconstruct(out int actualYear, out int actualMonth, out int actualDay);
```

Comment:
I'd use the deconstruction feature here to make it more obvious why we're doing this:

```csharp
var (actualYear, actualMonth, actualDay) = value;
```

Reply:
Yeah, I try doing that, but then I'd have to add the System.ValueTuple package, and I wasn't sure I could.

And in that case, to which project should I add it? Just to the test project?

Reply:
Yup - it's fine to have it in tests, but I don't want it as a dependency of Noda Time itself. (I believe fairly soon we won't need it at all for deconstruction to variables - I can find the Roslyn issue if you want.)